base/applications/mspaint/winproc.cpp

```diff
@@ -841,12 +840,20 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
         }
         case IDM_EDITCOPYTO:
         {
-            WCHAR szFileName[MAX_LONG_PATH] = L"*.png";
+            WCHAR szFileName[MAX_LONG_PATH];
+            CStringW strUntitled(MAKEINTRESOURCEW(IDS_DEFAULTFILENAME));
+            StringCchCopyW(szFileName, _countof(szFileName), strUntitled);
```

Comment:
I wonder whether, with some carefully chosen CStringW construction, we could avoid using two separate buffers here (strUntitled and szFileName), and instead manage to have only one (szFileName with correct size) that could also be initialized to the default name, before passing it to GetSaveFileName().

Reply:
Used `LoadStringW`.